src/components/Modal_Components/ManualTransaction.jsx

```diff
@@ -139,7 +146,8 @@ export const AddTransaction = props => {
             editBlock: state.editBlock,
             customBlock: state.customBlock,
             cat,
-            classes
+            classes,
+            errorMessage: "Create a valid name and a budget number over 0"
```

Comment:
Awesome job at refactoring here. We reduced the amount of views by just adding an error param to one of the views.